tools/export/iar.py

```diff
@@ -76,6 +76,8 @@ def generate(self, progen_build=False):
         project_data['tool_specific']['iar'].setdefault("misc", {})
         project_data['tool_specific']['iar'].update(tool_specific['iar'])
         project_data['tool_specific']['iar']['misc'].update(self.progen_flags)
+        project_data['tool_specific']['iar']['misc']['asm_flags'].remove(
+            self.toolchain.get_config_option(self.config_header)[0])
```

Comment:
Is there a way to provide configuration options to the IAR assembler in the exporters now? On the command line, it's done by switching to `-D`s instead of using `--preinclude`:

https://github.com/ARMmbed/mbed-os/blob/master/tools/toolchains/iar.py#L155